core/src/bufr/BufrReader/Exports/Variables/RemappedBrightnessTemperatureVariable.cpp

```diff
@@ -75,32 +86,89 @@ namespace bufr {
 
         // Get field-of-view number
         std::vector<int> fovn(fovnObj->size(), DataObject<int>::missingValue());
-        for (size_t idx = 0; idx < fovnObj->size(); idx++)
-        {
-           fovn[idx] = fovnObj->getAsInt(idx);
-        }
+        fovn = std::dynamic_pointer_cast<DataObject<int>>(fovnObj)->getRawData();
```

Comment:
Do this
`auto fovn = std::dynamic_pointer_cast<DataObject<int>>(fovnObj)->getRawData();`